code-of-conduct.md

```diff
@@ -2,47 +2,48 @@
+Reports should be submitted in writing (e.g. on Slack or via denhac.org email) to any member of our Board of Directors, listed at https://denhac.org/wiki/the-board; the entire board is also available via email at board@denhac.org. If your report is given to a single member of the board, they will ask to pull in a second board member of your choice, to share the detailed context of the issue in order to better represent it to the rest of the board and help address the issue.
 
 # What happens when a harassment report is submitted
 Report information will be shared and discussed privately over board-only slack and/or private board executive session - current board members are the only ones present at the time of discussion, but future board members may have access to the discussion log as well. Board members will investigate the report privately (potentially including use of our camera footage but we will check in with the reporter before taking any actions which might reidentify the reporter to a harasser), discuss and decide on next steps to address the incident, and communicate back to the reporter on the next steps chosen within a week after the next Executive Session completes. Anonymous summaries of public events that require a statement, and/or summary statistics around incident management, may also be shared more widely across denhac for transparency.
 
 # Potential consequences of violating the code of conduct
-denhac reserves the right to terminate any membership at any time for any reason, but we strive for the process to be as fair and open as privacy and discretion allow us to be. Upon breaching of the denhac rules and member agreements, the following actions may be performed in any order:
+denhac reserves the right to terminate any membership at any time for any reason, but the Board of Directors strives for the process to be as fair and open as privacy and discretion allow us to be. Upon breaching of the denhac rules and member agreements, the following actions may be performed in any order:
 
 * The member will get a verbal warning from a Director or a Manager.
 * The member will be called before the Board to explain and/or discuss the behavior in question, usually within Executive Session.
 * The member will receive a formal Warning, via letter or email, signed by a board member.
-* The Member will be put on probation.
-* Membership will be suspended for a length of time.
-* Membership will be terminated.
+* The member will be put on probation and removed from good standing.
+* Membership will be suspended for a predefined length of time, often until the next public board meeting. This can be done by any one board member or manager if there is a safety concern.
```

Comment:
live feedback: replace with link to bylaws sections which already define this, rather than duplicating